kong/plugins/aws-lambda/schema.lua

```diff
@@ -57,7 +40,6 @@ return {
         { aws_region = {
           type = "string",
           required = true,
-          one_of = REGIONS
```

Comment:
should this be a type "typedefs.host"? see https://github.com/Kong/kong/blob/master/kong/db/schema/entities/services.lua#L34

Reply:
I don't think so, @Tieske. That's only a portion of the host.

Reply:
https://github.com/Kong/kong-plugin-aws-lambda/blob/master/kong/plugins/aws-lambda/handler.lua#L166

Reply:
I get that, but currently it is wide open, using the typedef at least limits is to sensible values

Reply:
done!